### DISCUSSION TITLE
question: using Radicale with Thunderbird?

### DISCUSSION THREAD

**Discussion by monperrus** [Kozea/Radicale#1401] [Q&A client]
Hi all,

What Thunderbird plugin do you recommend to connect to a Radicale server?

**Comment by tberger2** [comment]
I use Lightning with Inverse SOgo Connector

**Comment by daks** [comment]
me too

**Comment by jaesivsm** [comment]
For calendar alone Lightning will do just fine.

For contact I used the sogo connector for a while but saw around here that [Cardbook](https://addons.thunderbird.net/en-GB/thunderbird/addon/cardbook/) was a much better contact server. I've been on it ever since.

**Comment by mrwsl** [comment]
Used the SOgo Connector as well but now I switched to Cardbook which seems much nicer and works flawless for me

**Comment by monperrus** [comment]
Thanks a lot, as I'm interested in contacts, I'll look into Cardbook.

**Comment by pbiering** [comment]
https://github.com/Kozea/Radicale/wiki/Client-Thunderbird
